db/migrate/20151203230940_add_date_type_to_content_view_erratum_filter_rules.rb

```diff
@@ -0,0 +1,5 @@
+class AddDateTypeToContentViewErratumFilterRules < ActiveRecord::Migration
+  def change
+    add_column :katello_content_view_erratum_filter_rules, :date_type, :string, :default => "issued", :null => false
```

Comment:
@mccun934 may also have an opinion about this, as i may be wrong.  After looking at spacewalk-clone-by-date it looks like they actually default to 'issued'